with-facebook-pixel: new implementation with app folder (#49880)
vercel/next.js: examples/with-facebook-pixel/app/readme.txt

> Implementation only for Next 13 version or greater with the app folder

Reviewer: ```suggestion
Implementation for Next 13+ using the App Router.
```
